src/data/text/match_call_messages.h

```diff
@@ -0,0 +1,476 @@
+#define MCFLAVOR(name) {gMatchCallFlavorText_##name##_Strategy, \
```

Comment:
I can't read this macro name and not laugh

Reply:
lmao